### ISSUE TITLE
The pddl-parser seems to reject empty effects without conjunctions

### ISSUE THREAD

**Issue by rpgoldman** [KCL-Planning/VAL#21]
This definition failed to parse:
```
(:ACTION DIVERGENCE
          :PARAMETERS () ; (?P ?O)
          :PRECONDITION ()
          :EFFECT ())
```
but this one succeeded:
```
(:ACTION DIVERGENCE
          :PARAMETERS () ; (?P ?O)
          :PRECONDITION ()
          :EFFECT (AND))
```
If I understand the syntax correctly, an empty effect as in the first example, seems to be explicitly accepted.

**Comment by guicho271828** [comment]
looks like so.
```
<emptyOr(x)>::=()
<emptyOr(x)>::=x
<action-def body> ::= [:precondition <emptyOr (pre-GD)>]
                      [:effect <emptyOr (effect)>]
```

**Comment by DerekLong101** [comment]
Yes, quite right, they should be accepted. This is an outstanding issue that I should have dealt with long ago. I will try to get on top of it and let you know!


Cheers

Derek

________________________________

**Comment by guicho271828** [comment]
If you are struggling to take time for maintaining this software, you could consider inviting active people as the collaborators (merge/push access) so that the project keeps being healthy.

**Comment by guicho271828** [comment]
My branch now contains a fix to this issue. b047344

**Comment by guicho271828** [comment]
ping --- is this fixed?

**Comment by DerekLong101** [comment]
Thanks for the ping. I will deal with this - I am finding time to move a few things along. You could issue a pull request for your proposed fix.
